### PR TITLE
CLI args: support catalogAdd with multiple URIs

### PR DESCRIPTION
**Pull request by aledsage**

### CODE REVIEW

File: server-cli/src/test/java/org/apache/brooklyn/cli/CliTest.java [hunk 1]
```diff
@@ -59,6 +60,8 @@
 import org.apache.brooklyn.core.entity.factory.ApplicationBuilder;
 import org.apache.brooklyn.core.entity.trait.Startable;
 import org.apache.brooklyn.core.location.SimulatedLocation;
+import org.apache.brooklyn.core.mgmt.internal.LocalManagementContext;
+import org.apache.brooklyn.core.mgmt.internal.LocalManagementContextRegistry;
```
Comment (johnmccabe) [inline server-cli/src/test/java/org/apache/brooklyn/cli/CliTest.java]: Did you miss this out from the commit?

Comment (aledsage) [inline server-cli/src/test/java/org/apache/brooklyn/cli/CliTest.java]: Yes, doh! Have added it and pushed again.

**Comment by johnmccabe** [comment]
Built and run Brooklyn and Clocker and was able to start with 5 catalogs loading as expected, lovely stuff, thanks @aledsage.

Will pull updated commit just to sanity check the unit tests now

**Comment by johnmccabe** [comment]
LGTM checked catalogInitial with catalogAdd, working as expected.

**Comment by johnmccabe** [comment]
Also checked the some persistance behaviours.
- Restarting with passing an unchanged bom is ok
- Retarting with a newer or older version than persisted is ok, the new/old versions get added and all expected versions are visible in the catalog
- Restarting with  an existing version whose contents have changed (ie change the name) puts the server in an error state - expected
- Restarting above errored system with the name reverted is ok, starts successfully
- Alternatively, restarting the errored system with --catalogForce sees it start successfully and the version you've altered gets added to the catalog overriding the persisted version (all other versions are still present)

**Comment by aledsage** [comment]
Thanks @johnmccabe , including for the through testing of restart scenarios! Merging now.

**Pull request merged**